### PR TITLE
stat: refresh page

### PR DESCRIPTION
**Pull request by iridacea**
Changes from the suggestions in #9346

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages/common/stat.md [hunk 1]
```diff
@@ -3,26 +3,26 @@
...
 > More information: <https://www.gnu.org/software/coreutils/manual/html_node/stat-invocation.html>.
 
-- Show file properties such as size, permissions, creation and access dates among others:
+- Show a specific file properties such as size, permissions, creation and access dates among others:
 
 `stat {{path/to/file}}`
 
 - Same as above but in a more concise way:
 
 `stat -t {{path/to/file}}`
```
Comment (navarroaxel) [inline pages/common/stat.md:12]:
```suggestion
`stat --terse {{path/to/file}}`
```

File: pages/common/stat.md [hunk 2]
```diff
@@ -3,26 +3,26 @@
 > Display file and filesystem information.
 > More information: <https://www.gnu.org/software/coreutils/manual/html_node/stat-invocation.html>.
 
-- Show file properties such as size, permissions, creation and access dates among others:
+- Show a specific file properties such as size, permissions, creation and access dates among others:
 
 `stat {{path/to/file}}`
 
 - Same as above but in a more concise way:
```
Comment (navarroaxel) [inline pages/common/stat.md:10]:
```suggestion
- Display properties about a specific file such as size, permissions, creation and access dates among others without labels:
```

File: pages/common/stat.md [hunk 3]
```diff
@@ -3,26 +3,26 @@
...
 `stat -t {{path/to/file}}`
 
-- Show filesystem information:
+- Show a filesystem information:
 
 `stat -f {{path/to/file}}`
 
 - Show only octal file permissions:
 
 `stat -c "%a %n" {{path/to/file}}`
```
Comment (navarroaxel) [inline pages/common/stat.md:20]:
```suggestion
`stat --format="%a %n" {{path/to/file}}`
```

File: pages/common/stat.md [hunk 4]
```diff
@@ -3,26 +3,26 @@
...
 
-- Show owner and group of the file:
+- Show an owner and a group of a specific file:
 
 `stat -c "%U %G" {{path/to/file}}`
```
Comment (navarroaxel) [inline pages/common/stat.md:24]:
```suggestion
`stat --format="%U %G" {{path/to/file}}`
```

File: pages/common/stat.md [hunk 5]
```diff
@@ -3,26 +3,26 @@
...
 
-- Show the size of the file in bytes:
+- Show a size of a specific file in bytes:
 
 `stat -c "%s %n" {{path/to/file}}`
```
Comment (navarroaxel) [inline pages/common/stat.md:28]:
```suggestion
`stat --format="%s %n" {{path/to/file}}`
```

**Comment by iridacea** [comment]
@navarroaxel Applied all your suggestions.
@yutyo Why self-assigning this PR?

**Comment by yutyo** [comment]
> @navarroaxel Applied all your suggestions.
> @yutyo Why self-assigning this PR?

To accelerate review, no worries :)

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Review by CleanMachine1 — APPROVED** [review]

**Comment by CleanMachine1** [comment]
Thanks for the contribution

**Pull request merged**